cvat/apps/engine/management/commands/runperiodicjob.py

```diff
@@ -10,14 +10,15 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser: ArgumentParser) -> None:
         parser.add_argument("job_id", help="ID of the job to run")
+        parser.add_argument("job_args", nargs="*", help="Arguments to pass to the job")
 
     def handle(self, *args, **options):
         job_id = options["job_id"]
 
         for job_definition in settings.PERIODIC_RQ_JOBS:
             if job_definition["id"] == job_id:
                 job_func = import_string(job_definition["func"])
-                job_func()
+                job_func(*options["job_args"])
```

Comment:
I don't think this is right - you should be passing args from the job definition, not from the command line. Also, what about kwargs?